### PR TITLE
add module for tweet scraping and word cloud generation

### LINKED ISSUE #6: module fails due to wordcloud package non installable for python 3.11
The script and modules works fine on google collab or jupyter ipynb, 
but the following installations fail
- `pip install wordcloud` for python 3.11
- pycharm module installation fails for wordcloud module at line `from wordcloud import WordCloud` in scraper.py

After investigation, I found there is an issue with the [word_cloud](https://github.com/amueller/word_cloud) package itself and works only till older versions of python.

There is a workaround by manual installation with wheels from https://www.lfd.uci.edu/~gohlke/pythonlibs/#wordcloud as suggested [here](https://github.com/amueller/word_cloud/issues/637#issuecomment-865095385). But that would make this module installation challenging and add a manual dependency.

I have added the concern in [this](https://github.com/amueller/word_cloud/issues/637) issue and seems like they have already worked on the fix in this [PR](https://github.com/amueller/word_cloud/pull/697) merged to master, but it just hasn't made it into a release yet.

### PR DESCRIPTION
**Pull request by mratanusarkar**
moving the code from the ipynb into a python module.
but, still need to look for the possibilities for the following:

- [ ] breaking down the code into even smaller modules and functions.
- [ ] look for a more generalized naming scheme.
- [ ] some functions feel hardcoded and could be further generalized.
- [ ] reduce file content of scraper.py
- [ ] fix issue with the [word_cloud package](https://github.com/amueller/word_cloud)
- [x] add documentation

Fixing the word_cloud package issue would also close https://github.com/mratanusarkar/twitter-sentiment-analysis/issues/6

### CODE REVIEW

**Comment by mratanusarkar** [comment]
@dsaha21 I have moved the code from the ipynb into a python module.
But I feel, we still need to look for the possibilities for the following points mentioned in the PR.

**Comment by mratanusarkar** [comment]
also, there is a concern with the [word_cloud package](https://github.com/amueller/word_cloud), for which I have opened an issue here https://github.com/mratanusarkar/twitter-sentiment-analysis/issues/6

**Comment by dsaha21** [comment]
> @dsaha21 I have moved the code from the ipynb into a python module. But I feel, we still need to look for the possibilities for the following points mentioned in the PR.

ok cool. Will look upon it.

File: Runner/main.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from module.scraper import TweetScraper
+from module.generator import TwitterWordCloud
+
+# create helper objects
+tweet_scraper = TweetScraper()
```
Comment (dsaha21) [inline Runner/main.py:5]: If we can comment here what does scrapper is doing then user can easily understand @mratanusarkar

**Review by dsaha21 — COMMENTED** [review]
Have gone through. Seems ok to me. Will mention something if needed. Thank you

File: Runner/main.py (hunk 1 above)
Comment (mratanusarkar) [reply Runner/main.py:5]: scraper is scraping the tweets @dsaha21 
also, **line 5** is object creation for the TweetScraper class, which is clearly understood.

**Comment by mratanusarkar** [comment]
raising PR as this is a long-standing issue and no response from [word_cloud package](https://github.com/amueller/word_cloud)

**Review by mratanusarkar — COMMENTED** [review]
moving developed code and progress to main, and will resume once the [word_cloud package](https://github.com/amueller/word_cloud) issue is resolved by the package owner.

**Comment by mratanusarkar** [comment]
5/6 unfinished tasks will be taken up once the [issue #6](https://github.com/mratanusarkar/twitter-sentiment-analysis/issues/6) is resolved.

**Pull request merged**